bedrock/firefox/templates/firefox/welcome/page17/page17-c.html

```diff
@@ -18,22 +18,25 @@
 {% endblock %}
 
 {% if LANG == "de" %}
-  {% set page_title = 'Starte im neuen Jahr mit der Bild-im-Bild-Funktion durch' %}
-  {% set main_title = 'Starte im neuen Jahr mit der Bild-im-Bild-Funktion durch'|safe %}
+  {% set page_title = 'Surfe im neuen Jahr sicher durch das Internet und nutze dabei die Bild-im-Bild-Funktion' %}
+  {% set main_title = 'Surfe im neuen Jahr sicher durch das Internet und nutze dabei die Bild-im-Bild-Funktion.'|safe %}
+  {% set main_tagline = 'Mit der Bild-im-Bild-Funktion kannst du einfach mehrere Aufgaben gleichzeitig bearbeiten und dir die neuesten Tutorials oder Aufnahmen angucken, während du arbeitest.' %}
   {% set main_cta = 'Jetzt testen' %}
   {% set picto_one_title = 'Ein Meeting verfolgen und gleichzeitig Notizen machen' %}
   {% set picto_two_title = 'Ein Koch-Tutorial angucken, während du das Rezept anschaust' %}
   {% set picto_three_title = 'Die Kinder entertainen und nebenbei eine E-Mail fertig schreiben' %}
 {% elif LANG == "fr" %}
-  {% set page_title = 'Entamez une année productive avec la fonctionnalité plébiscitée d’incrustation vidéo' %}
-  {% set main_title = 'Entamez une année productive avec la fonctionnalité plébiscitée d’incrustation vidéo'|safe %}
+  {% set page_title = 'Mettez cette année sous le signe de la productivité avec l’incrustation vidéo' %}
+  {% set main_title = 'Mettez cette année sous le signe de la productivité avec l’incrustation vidéo'|safe %}
+  {% set main_tagline = 'Faites plusieurs choses à la fois facilement avec l’incrustation vidéo: visionnez des tutoriels, des enregistrements et plus encore tout en travaillant.' %}
   {% set main_cta = 'Essayez-le dès maintenant' %}
```

Comment:
**Nit:** This needs to be updated to `L’essayer maintenant` as seen here: https://docs.google.com/spreadsheets/d/1rFadFRe0X4D2mKKjUE2xJfqZr6fzjyD_2DR9E1VN-lQ/edit#gid=455235426